frontend/src/container/SideNav/menuItems.tsx

```diff
@@ -91,7 +97,6 @@ const menuItems: SidebarItem[] = [
 		key: ROUTES.MESSAGING_QUEUES,
 		label: 'Messaging Queues',
 		icon: <ListMinus size={16} />,
-		isBeta: true,
```

Comment:
Please check this; I didn't get any update regarding changing the Messaging Queue to non-beta